Components documentation improvements (#2121)
withastro/starlight: docs/src/content/docs/components/link-cards.mdx

---
title: Link Cards

> description: Learn how to display links prominently with the LinkCard component.

Reviewer: ```suggestion
description: Learn how to display links prominently as cards in Starlight.
```
